crates/core/src/test.rs

```diff
@@ -14330,6 +14330,40 @@ fn ruby_class_2() {
     .unwrap();
 }
 
+#[test]
+fn ruby_class_method() {
+    run_test_expected({
+        TestArgExpected {
+            pattern: r#"
+                |language ruby
+                |
+                |`class Box
+                |   $methods
```

Comment:
We also want to enable searching for the method by itself:

Ex.

```
`def $name($_)
    $body
end` where {
  $name => `foo`
}
```

Reply:
added another test with that check in https://github.com/getgrit/gritql/pull/286/commits/30e5f6b56961b9f8d31fcc1159df7a9c65f032d1#diff-041f066ed81994bf9c7f0a0f795e5c85faab0c47d152231fce8b15d7bd2aa45aR14367-R14404